App/Downloader.php

```diff
@@ -241,40 +222,39 @@ protected function setFilters()
         return true;
     }
 
-    private function setSeriesFilter($options)
+    private function setSeriesFilter($options): void
     {
         if (isset($options['s']) || isset($options['series-name'])) {
-            $series = isset($options['s']) ? $options['s'] : $options['series-name'];
+            $series = $options['s'] ?? $options['series-name'];
 
-            if (! is_array($series))
+            if (! is_array($series)) {
                 $series = [$series];
+            }
 
-            $slugify = new Slugify();
+            $slugify = new Slugify;
             $slugify->addRule("'", '');
 
-            $this->filters['series'] = array_map(function($serie) use ($slugify) {
-                return $slugify->slugify($serie);
-            }, $series);
+            $this->filters['series'] = array_map(fn ($serie): string => $slugify->slugify($serie), $series);
 
-            Utils::write(sprintf("Series names provided: %s", json_encode($this->filters['series'])));
+            Utils::write(sprintf('Series names provided: %s', json_encode($this->filters['series'])));
         }
     }
 
-    private function setEpisodesFilter($options)
+    private function setEpisodesFilter($options): void
     {
         $this->filters['episodes'] = [];
 
         if (isset($options['e']) || isset($options['series-episodes'])) {
-            $episodes = isset($options['e']) ? $options['e'] : $options['series-episodes'];
+            $episodes = $options['e'] ?? $options['series-episodes'];
 
-            Utils::write(sprintf("Episode numbers provided: %s", json_encode($episodes)));
+            Utils::write(sprintf('Episode numbers provided: %s', json_encode($episodes)));
 
             if (! is_array($episodes)) {
                 $episodes = [$episodes];
             }
 
             foreach ($episodes as $episode) {
-                $positions = explode(',', $episode);
+                $positions = explode(',', (string) $episode);
```

Comment:
it's int

Reply:
may be we cast the returned values since explode is a string function.